src/Dfe.Spi.UkrlpAdapter.Infrastructure.UkrlpSoapApi/SoapException.cs

```diff
@@ -0,0 +1,13 @@
+namespace Dfe.Spi.UkrlpAdapter.Infrastructure.UkrlpSoapApi
+{
+    public class SoapException : UkrlpSoapApiException
+    {
+        public string FaultCode { get; }
+
+        public SoapException(string faultCode, string faultString)
+            : base(faultString)
+        {
+            FaultCode = faultCode;
+        }
+    }
+}
```

Comment:
Have we not done stuff with SOAP elsewhere, or is this the same moved about/refectored? If so, couldn't this be common-ised?

Reply:
It was copy/pasted from GIAS adapter. I did contemplate it, but am hoping it only these two so decided not to. Do you see anywhere else we will use it?

Reply:
¯\_(ツ)_/¯ Not really, at this point, generally if something is used in more than one place I'd be tempted to common-ise it, even if it is just 2 places. In theory, if we were to develop more adapters in the future that use SOAP, it could be built on top of this stuff. I'm not going to lose sleep over this though!

I'll resolve this.